mscore/musescore.cpp

```diff
@@ -1995,8 +1996,14 @@ MuseScore::MuseScore()
       connect(this, SIGNAL(musescoreWindowWasShown()), this, SLOT(checkForUpdates()),
             Qt::ConnectionType(Qt::QueuedConnection | Qt::UniqueConnection));
 
-      if (!converterMode && !pluginMode)
+      if (!converterMode && !pluginMode) {
             _loginManager = new LoginManager(getAction(saveOnlineMenuItem), this);
+            const bool loadSuccess = _loginManager->load();
+
+            if (cliSaveOnline && !loadSuccess) {
+                  qFatal(qUtf8Printable(tr("No login creditials stored. Please sign-in via the GUI.")));
```

Comment:
I have tried building `v3.6_public_beta` with Nixpkgs and it failed at this line with the error:
```
/build/source/mscore/musescore.cpp: In constructor 'Ms::MuseScore::MuseScore()':
/build/source/mscore/musescore.cpp:2004:104: error: format not a string literal and no format arguments [-Werror=format-security]
 2004 |                   qFatal(qUtf8Printable(tr("No login credentials stored. Please sign in via the GUI.")));
      |                                                                                                        ^
```
(Nixpkgs by default builds with `-Wformat -Wformat-security -Werror=format-security`.)

Reply:
And that is the only place where it complains about this?

Reply:
Yes, 3.5.2 used to build with format-security, and 3.x builds with the fix in #7118.

Reply:
Continued at https://github.com/musescore/MuseScore/pull/7118#discussion_r546260550